### PR TITLE
style(component): fix styles for Project Header

### PR DESCRIPTION
**Pull request by theodesp**
## Screenshots

Screenshot 2022-03-23 at 16 49 20

Screenshot 2022-03-23 at 16 48 50

### CODE REVIEW

File: src/components/TaxonomyTerms/TaxonomyTerms.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import styles from './TaxonomyTerms.module.scss';
 
 export default function TaxonomyTerms({ post, taxonomy }) {
   const termLinks = post?.[taxonomy]?.()?.edges.map((edge, index) => {
+    // eslint-disable-next-line no-unsafe-optional-chaining
```
Comment (apmatthews) [inline src/components/TaxonomyTerms/TaxonomyTerms.js:7]: There is no optional chaining being done in the next line. Do we need this?

File: src/hooks/useSearch.js [hunk 2]
```diff
@@ -138,7 +138,8 @@ export default function useSearch() {
 
     const res = await fetchResults(debouncedSearchQuery, pageInfo?.endCursor);
 
-    setSearchResults((prev) => uniqBy([...prev, ...res.nodes], (v) => v.id));
+    // eslint-disable-next-line no-unsafe-optional-chaining
```
Comment (apmatthews) [inline src/hooks/useSearch.js:141]: I resolved this differently in #43. What is the reasoning behind disabling the rule here as opposed to removing the optional chaining?

File: src/components/TaxonomyTerms/TaxonomyTerms.js (hunk 1 above)
Comment (theodesp) [reply src/components/TaxonomyTerms/TaxonomyTerms.js:7]: Sorry wrong place

**Comment by apmatthews** [comment]
Looks great! One more tiny detail is that the left margin on the `figure` throws the project header a little off center of the content. It's most obvious in this screenshot, but still has an effect on desktop as well.
Screen Shot 2022-03-23 at 3 51 04 PM

**Comment by theodesp** [comment]
> Looks great! One more tiny detail is that the left margin on the `figure` throws the project header a little off center of the content. It's most obvious in this screenshot, but still has an effect on desktop as well. Screen Shot 2022-03-23 at 3 51 04 PM

Resolved thank you.

**Review by apmatthews — APPROVED** [review]
Beautiful 👌🏻 Thanks for rolling with the margin updates!

**Pull request merged**